modules/tensor_mechanics/doc/content/modules/tensor_mechanics/1d_elastic_waves.md

```diff
@@ -1,101 +1,127 @@
-
-!alert note
-One can verify that $r e^{i(\omega t - k x)}$ is a solution for [eq3]$\Longleftrightarrow$ $k^2= \frac{\omega^2}{c_p^2}$ by direct substitution.
-
-Now, [eq2] and the associated BCs. and can be written as,
+The boundary conditions also need to be converted to the frequency domain through a Fourier transform.  A time harmonic Neumann BC given by
 \begin{equation}
-     -\nabla \cdot \sigma- \omega^2 u = 0 \label{eq4}
+    f= A \, cos(2 \pi f t) \qquad \text{at} \, x=L
+    \label{eq4}
 \end{equation}
-
+with amplitude $A$ and frequency $f$, is transformed to
 \begin{equation}
-    \nabla u= ik \, u \, \qquad \text{at} \, x=0,
-    \\
-    u= A \, cos(2 \pi f t) \, \qquad \text{at} \, x=L,\label{eq5}
+    F= A \, \qquad \text{at} \, x=L
+    \label{eq5}
 \end{equation}
 
-Transforming [eq4] and the associated boundary conditions in [eq5] in time to the frequency domain, we obtain:
-\begin{equation}
-     -\nabla \cdot \hat \sigma- \omega^2  U = 0 \label{eq6}
-\end{equation}
-with the following boundary conditions:
+# Cantilever Beam Example
+
+The cantilever beam shown in [cantilever] is subjected to a time harmonic force on the right side in the out-of-plane and vertical directions.  In this example the frequency of the time varying load is swept over a range.  The displacement at the midpoint of the beam is recorded at each frequency.  This type of output, displacement as a function of frequency, is a frequency response function (FRF) or transfer function.  Displacement peaks in the FRF indicate natural frequencies.  
+
+!media media/tensor_mechanics/Cantilever_beam.png style=width:60%; caption=2D cantilever problem with a prescribed displacement boundary condition on the right end. id=cantilever
+
+The analytic solution for the free vibration of a cantilever [Euler Bernoulli beam](https://en.wikipedia.org/wiki/Euler%E2%80%93Bernoulli_beam_theory).  The analytic eigenvalues, $\omega_n$, are given by
 \begin{equation}
-    \nabla U= ik \, U \, \qquad \text{at} \, x=0,
-    \\
-U= 0.5 \, \qquad \text{at} \, x=L,\label{eq7}
+  \omega_n=k^2_n\sqrt\frac{EI^2}{\rho A L^4}
 \end{equation}
+where $I$ is the moment of inertia, $A$ the cross sectional area, $L$ the beam length, and $k_n$ are the wave numbers.
+For a cantilever beam, the first three wave numbers, $k_n$, are\\
+$k_1=1.875$\\
+$k_2=4.694$\\
+$k_3=7.855$\\
+The moment of inertia for a rectangular cross section beam is $I=\frac{wh^3}{12}$, where $h$ is the dimension in the direction being loaded and $w$ is the other cross sectional dimension.  
+
+For an aluminum cantilever beam, $E=$68e9 Pa, $\nu=$0.36, $\rho=$27.e3 kg/m$^3$.  The dimensions of the cantilever beam are $L=$1m with cross section dimensions $a=$0.1m and $b=$0.15m.
+The analytic first and second natural frequencies for this system bending in directions transverse to the beam axis are are:\\
```

Comment:
I do not understand this "for this system bending in directions transverse to the beam axis are are", may be rephrasing it.

Reply:
reworded